lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -2,8 +2,18 @@
 
 const BbPromise = require('bluebird');
 const async = require('async');
+const path = require('path');
 
 module.exports = {
+  writeCreateTemplateToDisk() {
+    const cfTemplateFilePath = path.join(this.serverless.config.servicePath,
+      '.serverless', `cf-template-create-${(new Date).getTime().toString()}.json`);
```

Comment:
Imho timestamp is good here so it doesn't overwrite anything in the future in case we change this implementation a bit and keep more things around.